tests/testthat/test-printing.R

```diff
@@ -146,8 +146,12 @@ test_that("newline in column names and possibly cell values work", {
     matform2 <- matrix_form(tbl2)
     expect_identical(dim(matform2$strings),
                      c(18L, 5L))
-    expect_identical(attr(matform2, "nlines_header"),
-                     4L)
+    # expect_identical(attr(matform2, "nlines_header"),
+    #                  4L)
+    expect_identical(attr(matform2, "nrow_header"),
+                     3L)
+    expect_identical(attr(matform2, "ncols"),
+                     4)
```

Comment:
@gmbecker, the commented line is failing as there is not such attribute anymore. I think the name of it was changed to `nrow_header`, but its associated value is `3L` and not `4L`. So, just to show, I added the variable that has 4 as value (not `4L` btw). Checking the tested matrix though, I do not know what is the number of rows we expect from the header (I would say four??). Here, it is:
```
Ethnicity               Arm                 Arm       
Factor2                    1                   2        
                    Male     Female     Male     Female 
                   (N=256)   (N=248)   (N=248)   (N=248)
————————————————————————————————————————————————————————
Caucasian                                               
  Level A                                               
    mean            56.42     55.57     56.19     54.53 
    median          55.91     55.42     58.40     51.73 
  Level B                                               
    mean            54.28     55.24     54.22     54.92 
    median          54.71     55.47     54.96     55.17 
African American                                        
  Level A                                               
    mean            55.77     55.33     56.26     54.30 
    median          55.06     54.39     57.20     53.94 
  Level B                                               
    mean            53.85     56.55     56.80     55.88 
    median          54.36     57.21     57.39     55.71 
```

Reply:
Unrelated and maybe not an issue at all but I noticed an indentation on the first `topleft` name and not on the second. As I do not see any indentation directive in the test, is it a designed standard? Here a screenshot of it printed to console:
![Screenshot 2022-11-17 163212](https://user-images.githubusercontent.com/11279768/202489501-b4806751-8f6f-424e-b219-9851defc8bd1.png)

Reply:
nrow header and nlines_header have always been different. 

It is true that the attribute no longer exists, but the getter still does, and should have been used anyway (`mf_nlheader(matform)`)

In fact we need to search out all calls to attr and get rid of most of them as they penetrate the abstraction

Reply:
Using search it seems like this was the only case where `attr(` is used, so no problem!